jetty-ee10/jetty-ee10-jaspi/src/main/java/org/eclipse/jetty/ee10/security/jaspi/JaspiMessageInfo.java

```diff
@@ -101,69 +100,54 @@ public void setResponseMessage(Object response)
         _response = ServletContextResponse.getServletContextResponse((ServletResponse)response);
     }
 
-    public String getAuthMethod()
+    public String getAuthenticatorName()
     {
-        return _map.getAuthMethod();
-    }
-
-    public boolean isAuthMandatory()
-    {
-        return _map.isAuthMandatory();
+        return _map.getAuthenticatorName();
```

Comment:
This method is never used.
`_map.getAuthenticatorName()` is only ever used here.
I think we can totally remove the field (and the logic) from `MIMap`, and that should also allow to remove the delegation by removing `MIMap` entirely, which in turn fixes the TODO present in the `MIMap` class (and hopefully also fixes the use of generics for the Map?).

Reply:
I've removed the unused method, but for any other jaspi cleanup, I'd prefer to punt that to a new issue/PR.    I think there is a lot that needs careful thought about JASPI, so I'd prefer not to delay this PR whilst I page in what the hell JASPI is again and do a clean up.